README.md

```diff
@@ -50,3 +50,164 @@ FAQ
 ---
 
 Go to the [FAQ](https://github.com/Elao/symfony-standard/wiki/FAQ)
+
+Install ElaoStrap (optionnal)
+---
+
+Install our custom twitter bootstrap Symfony bundle : [ElaoStrap](https://github.com/Elao/ElaoThemeElaoStrapBundle)
```

Comment:
C'est pas vraiment un "custom twitter bootstrap symfony bundle', mais plutôt un "ElaoStrap theme bundle for Symfony"